wgpu/Cargo.toml

```diff
@@ -197,19 +148,47 @@ serde = { workspace = true, features = ["default", "derive"], optional = true }
 smallvec.workspace = true
 static_assertions.workspace = true
 
-[dependencies.naga]
-workspace = true
-optional = true
+[target.'cfg(any(target_arch = "wasm32", not(target_os = "emscripten")))'.dependencies]
+js-sys = { workspace = true, features = ["default"] }
```

Comment:
This doesn't belong in the next section?

Reply:
Oh shoot, this was actually a test that was left in. Will fix